app/templates/session-expired.html

```diff
@@ -12,7 +12,8 @@ <h1 class="saturn" {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>{{
   <div class="panel panel--simple panel--success panel--spacious u-mb-s">
     <p>{{ _("To help protect your information we have signed you out") }}.</p>
   </div>
-
+  {% if respondent_account_url is not none %}
```

Comment:
Here also same as above, this would be true for an empty string, so just `{% if respondent_account_url %}` would cover both cases.